arangod/SystemMonitor/AsyncRegistry/PrettyPrinter/README.md

```diff
@@ -16,10 +16,19 @@ This ensures that the pretty-printer is loaded to gdb. You can check if it is lo
 
 ### Run tests
 
-Inside src-folder run unittests via
+#### Unit tests
+
+```
+ctest --build -R async_registry_python_pretty_printer_test
```

Comment:
Have we used this before? Will it be executed in the CI?

Reply:
No, unfortunately this currently does not run in the CI, this is still on my todo list. It requires some additions to `./circleci/generate_config.py`.
We need to add a ctest-job in the CI and need to define somewhere - possibly in a file - which ctests should be executed (similar to our tests/test-definitions.txt file). I guess adding the job and parsing this new file needs to be done in `./circleci/generate_config.py`.

Reply:
But at least we can now execute the tests locally (as described in the readme) to see if we broke the pretty printer.